app/src/main/java/com/capstone/event_finder/activities/SignUpActivity.java

```diff
@@ -198,7 +213,7 @@ private void signUpUser() {
         user.put("event_categories", categoryListOfStrings);
         user.put("event_categories_string", allInterestCategories);
         user.put("profile_image", file);
-
+        user.put("private", isPrivate);
```

Comment:
❓ What happens to existing bookmarks when the user changes their privacy?  Should we be looping through all the user's bookmarks and updating their ACL?  (Remember we can't _completely_ rely on the client-side check you're making in `PosterProfileActivity`)